src/components/TransactionsTab/TransactionsTab.styles.tsx

```diff
@@ -108,6 +114,30 @@ export const LegendLine = styled.span`
   background: transparent;
 `;
 
+export const StyledTooltip = styled(Tooltip)<{ $isTooltip: boolean }>`
+  display: ${({ $isTooltip }) => ($isTooltip ? "flex" : "none")};
+  position: relative;
+  z-index: 3;
+  max-width: min-content;
+  margin-top: -2rem;
+  top: 1.25rem;
+  margin-left: 68%;
```

Comment:
`margin-left: 68%;`

Don't use esoteric values like this.

Reply:
switched to 15rem